ProcessMaker/processmaker
# FOUR-19753 : The task comments is not showing the buttons to register the comments

## Issue & Reproduction Steps
1. Open a task ->  /task
2. Go to comments tab
3. Tried to add a comments

## Solution
- Change the css styles in the view

## How to Test
1. Open a task
2. Verify the tab comments and tried to add a comment

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-19753

## Code Review Checklist

ci:deploy
